class-newspack-blocks.php

```diff
@@ -361,19 +361,14 @@ function ( $acc, $block ) use ( $block_name ) {
 			$args['post__in'] = $specific_posts;
 			$args['orderby']  = 'post__in';
 		} else {
-			$args['posts_per_page'] = $posts_to_show;
-			if ( ! self::is_experimental_mode() ) {
-				$args['posts_per_page'] += count( $newspack_blocks_post_id );
-			}
+			$args['posts_per_page'] = $posts_to_show + count( $newspack_blocks_post_id );
```

Comment:
```suggestion
			$args['posts_per_page'] = $posts_to_show;
```

I think the logic should be the other way round: we were adding the count if it was **not** experimental

Reply:
Oops, quite correct! Fixed in https://github.com/Automattic/newspack-blocks/pull/530/commits/19c7a6c5946eb9cbd60e838f6b9b9db06dda0855